proto/elys/tradeshield/params.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package elys.tradeshield;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/elys-network/elys/x/tradeshield/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+  
+}
```

Comment:
Let's not keep Params if there is no need for it. No query or anything requires as it's empty

Reply:
Kept it because we may need it to store max orders, or other params for example `history_number1 etc